src/profile/components/multiItemRow/multiItemRow.module.css

```diff
@@ -0,0 +1,43 @@
+.rowContentWrapper {
+  background: var(--color-black-ultra-light);
+  margin: 0 0 var(--spacing-m) 0;
+  padding: var(--spacing-m);
+}
+
+.rowContentWrapper form {
+  width: 100%;
+}
+
+.editableRow {
+  display: flex;
+  flex-direction: column;
+}
+
+/* fixes bug in HDS TextInput when it is wrapped to an element with display:flex: */
```

Comment:
There might be a fix coming to this in the next release of HDS 😅 . We removed TextInput input wrapper height 100%, which might be related to this.

Reply:
The fix should now be available: https://www.npmjs.com/package/hds-react

Reply:
NumberInput, PhoneInput and other HDS fixes mentioned here were released after this was originally created. HDS update would probably cause a lot of changes, so did not want to update it in this PR and make it larger. I'd rather make a new ticket for HDS updates.

Reply:
HDS update ticket created https://helsinkisolutionoffice.atlassian.net/browse/HP-727